samples/CallWithChat/src/app/views/CallScreen.tsx

```diff
@@ -111,7 +112,6 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
       rtl={currentRtl}
       joinInvitationURL={window.location.href}
       formFactor={isMobileSession ? 'mobile' : 'desktop'}
-      /* @conditional-compile-remove(call-readiness) */
```

Comment:
should this have been removed?

Reply:
this should be here for the callReadiness opt in work Caroline did I believe